tox.ini

```diff
@@ -103,19 +103,50 @@ usedevelop = False
 skip_install = True
 # don't install any Python dist deps
 deps =
-  pbr
-#  setuptools_scm
+    setuptools_scm
 # reset pre-commands
 commands_pre =
 # build the docker container
 commands =
     sh -c '\
-    TAG=$(python -c "from pbr.packaging import get_version; print(get_version(\".\"))") && \
-    #TAG=$(python -c "from setuptools_scm import get_version; print(get_version())") && \
+    TAG=$(python -c "from setuptools_scm import get_version; print(get_version())") && \
     echo Building version "$TAG"... && \
     sudo docker build \
         --pull --rm \
         {posargs:-t ansible/molecule:$TAG} .\
     '
 whitelist_externals =
     sh
+
+
+[testenv:build-dists-local]
+description =
+    Generate dists which may be not ready
+    for upload to PyPI because of
+    containing PEP440 local version part
+# `usedevelop = true` overrides `skip_install` instruction, it's unwanted
+usedevelop = false
+# don't install octomachinery itself in this env
```

Comment:
`octomachinery`?

Reply:
:D 
that's from my frameworks for writing GitHub Apps and Actions...
just artifacts of copy-pasting from different places. i'll clean that up